Dockerfile

```diff
@@ -2,11 +2,11 @@
 FROM node:12 as builder
```

Comment:
Yes, trivy scanning was done in this image by myself and Yemi independently. Dockerfile has been replaced already with new one in weekend but this is still valid in Docker.base file which is initial config for base image. 
Reason:
No vulnerability with this image. This image has been used by element as their base image. I understand we want to bring this version to at least 14.x.x version which is latest stable version. I am not that confident to use 15.x.x version yet as last time I tried to use this , it tries to install chocolaty as well and crashed without installing npm. Node should ideally install npm packages as well. Anyways, this dicussion about 15.x. Yemi and I had discussed we should try to use latest. This would be just initially getting something working on browser on given default image.Within few days, Yemi has planned to try to create an additional image with 14.x version based on our chat and maybe have two images in docker.io registry. One is default 12th version of node, other would have a different image with 14.x version of node. We discussed that 14th version if gets in problem to run our project then we roll back to 12th version in emergency situation. Again, 12th version did not have issues.